File: client/src/main/java/at/uibk/dps/dml/client/NodeLocation.java
package at.uibk.dps.dml.client;

public class NodeLocation {

    private final String region;

    private final String accessPoint;

    private final String provider;

    public NodeLocation(String region, String accessPoint, String provider) {
        this.region = region;
        this.accessPoint = accessPoint;
        this.provider = provider;
    }

    public String getRegion() {
        return region;
    }

    public String getProvider() {
        return provider;
    }

    public String getAccessPoint() {
        return accessPoint;
    }

}

File: client-py/dml/util/NodeLocation.py
class NodeLocation:
    """
    Node Location
    """

    def __init__(self, region: str, access_point: str, provider: str):
        self.region = region
        self.access_point = access_point
        self.provider = provider

    def get_region(self) -> str:
        return self.region

    def get_access_point(self) -> str:
        return self.access_point

    def get_provider(self) -> str:
        return self.provider
